packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -178,15 +164,11 @@ class InteractiveGraphEditor extends React.Component<Props> {
     displayName = "InteractiveGraphEditor";
     className = "perseus-widget-interactive-graph";
 
-    static defaultProps: DefaultProps = {
-        ...InteractiveGraph.defaultProps,
+    static defaultProps: InteractiveGraphDefaultWidgetOptions & {
+        valid: boolean;
+    } = {
+        ...interactiveGraphLogic.defaultWidgetOptions,
```

Comment:
I think this might be a mistake from a different PR. `defaultProps` for all the other widget editors is a subset of widget options - which makes sense since the editors are for generating widget options. This one uses `valid` though, which isn't part of `PerseusInteractiveGraphWidgetOptions`.

When doing these kind of large-scale reorgs, I try not to be too opinionated; so I kept it in a way that I think highlights that it's not part of widget options.

Reply:
I bet it's state again. However, isn't it `boolean | string`?

<img width="560" alt="image" src="https://github.com/user-attachments/assets/fc5874f7-31a8-4eb8-9a28-364ddfe33e46" />

Reply:
I think it originates from the `InteractiveGraphSettings` component: https://github.com/Khan/perseus/blob/5173c2e43bf939159f420dcd448b90691d52353b/packages/perseus-editor/src/widgets/interactive-graph-editor/components/interactive-graph-settings.tsx#L436

Reply:
That makes me think it's actually `valid: true | string` right?

> // a string message, not false